jest.config.cjs

```diff
@@ -1,4 +1,16 @@
 module.exports = {
-    preset: 'ts-jest',
-    testEnvironment: "jsdom",
-};
+  transform: {
+    '^.+\\.(t|j)sx?$': '@swc/jest',
+  },
+  testEnvironment: 'jsdom',
+  testEnvironmentOptions: {
+    customExportConditions: [''],
```

Comment:
Jest thinks jsdom is browser so they load the `browser` exports. That is fundamentally broken and one of the main reasons I'm recommending Vitest to everyone. If you can, consider Vitest. You will have much less pain in general when testing with it.

Reply:
thanks for the great background info, Vitest may be an option in the future.